src/Compilers/Core/Portable/Compilation/IOperation.cs

```diff
@@ -104,6 +104,10 @@ public enum OperationKind
         // Newly added
 
         ConditionalAccessExpression,
-        IncrementExpression
+        IncrementExpression,
+
+        Argument,
```

Comment:
Could you please show me an example of this? I did a quick search on [MSDN](https://msdn.microsoft.com/en-us/library/f25e2b6b.aspx), but all I got is "Every optional parameter in the procedure definition must specify a default value."